components/dashboard/dashboard.go

```diff
@@ -82,6 +83,10 @@ func (d *Dashboard) GetComponentName() string {
 	return ComponentNameUpstream
 }
 
+func (d *Dashboard) UpdateStatus(in *status.ComponentsStatus) {
+	in.Dashboard = &status.DashboardStatus{}
```

Comment:
Did not check too deep, why dashboard is special?

Reply:
Dashboard dont have an upstream for now. so keeping it empty for now